code.swift

```diff
@@ -54,27 +54,23 @@ class PeriodicTableViewController: UIViewController {
     }
     
     // This next method was added by a new member of the team. Identify the 10 formatting, naming, and logic issues below:
-    @IBAction func chooseRandomElement(_ sender: Any){
-        let ranElem = elementsContoller.elements.randomElement()
+    @IBAction func chooseRandomElement(_ sender: Any) { //spacing
+        var ranElem = elementsContoller.elements.randomElement() //this needs to be mutable.
```

Comment:
let is fine here, but something about this line doesn’t match how it’s done in the rest of the file...